next.config.js

```diff
@@ -2,7 +2,6 @@
 const nextConfig = {
   reactStrictMode: true,
   output: "export",
-  basePath: "/",
```

Comment:
No, this would break the integration in Github Pages

Reply:
How are you dealing with the following error then?

```
Error: Specified basePath /. basePath has to be either an empty string or a path prefix"
```

Reply:
Its only on development, you remove the line while you are in dev mode, then you put it back for prod

Reply:
Final solution should be this I guess:

```ts
basePath: process.env.NODE_ENV === "production" ? "/" : undefined
```